.github/workflows/go.yml

```diff
@@ -195,6 +193,17 @@ jobs:
           exclude-azure-developer-cli-credential: true
           exclude-interactive-browser-credential: true
 
+      - name: Update archives
+        if: startsWith(github.ref, 'refs/tags/v') # skip this step for snapshots because we don't know the name of the archive
```

Comment:
If we skips the archive update, why don't we skip the signing step too?

Reply:
I think we might want it for snapshot releases? I just realized that I can move the zip stuff into the next job and do it on Ubuntu, where I can use shell glob to update the zip